tests/Fixer/Operator/OperatorLinebreakFixerTest.php

```diff
@@ -190,6 +190,114 @@ function foo()
                 prepare_value:
                 $objectsPool[$value] = [$id = \count($objectsPool)];
         '];
+
+        yield 'handle ignored operator with -> at the beginning' => [
+            '<?php
+class ClassA
+{
+    public function foo($bar, $baz)
+    {
+        $a = $bar
+            || $baz;
+        return $this;
+    }
+
+    public function bar()
+    {
+        return $this;
+    }
+
+    public function baz()
+    {
+        return $this;
+    }
+}
+$class = new ClassA();
+$res = $class->foo()->
+    bar()->
+    baz();
+',
+            '<?php
+class ClassA
+{
+    public function foo($bar, $baz)
+    {
+        $a = $bar ||
+            $baz;
+        return $this;
+    }
+
+    public function bar()
+    {
+        return $this;
+    }
+
+    public function baz()
+    {
+        return $this;
+    }
+}
+$class = new ClassA();
+$res = $class->foo()->
+    bar()->
+    baz();
+',
+            ['ignored_operators' => ['->']],
+        ];
+
+        yield 'handle ignored operator with -> at the end' => [
+            '<?php
+class ClassA
+{
+    public function foo($bar, $baz)
+    {
+        $a = $bar
+            || $baz;
+        return $this;
+    }
+
+    public function bar()
+    {
+        return $this;
+    }
+
+    public function baz()
+    {
+        return $this;
+    }
+}
+$class = new ClassA();
+$res = $class->foo()
+    ->bar()
+    ->baz();
+',
+            '<?php
+class ClassA
+{
+    public function foo($bar, $baz)
+    {
+        $a = $bar ||
+            $baz;
+        return $this;
+    }
+
+    public function bar()
+    {
+        return $this;
+    }
+
+    public function baz()
+    {
+        return $this;
+    }
+}
+$class = new ClassA();
+$res = $class->foo()
+    ->bar()
+    ->baz();
+',
+            ['ignored_operators' => ['->']],
+        ];
```

Comment:
Let's add a test case with code like this:
```
$x = $a and
     $b
     AND $c;
```
and config `['ignored_operators' => ['And']]`.